Automattic/studio
# fix(sync): keep up to date url and syncSupport in local storage

## Related issues

- Fixes https://github.com/Automattic/dotcom-forge/issues/9816

## Proposed Changes

If user connected website and then something was changed on wordpress.com side - we don't reflect changes to appdata-v1.json.
With this PR we start doing it with `url` and `syncSupport` properties.
Later I will handle staging sites in the follow up PR.

## Testing Instructions
1. DON'T switch to this branch, it's necessary to do in 9-th step
1. Open wordpress.com
2. Purchase atomic website
3. Open Studio
4. Connect previously purchased website
5. Open `appdata-v1.json` and assert that you see:
	1. "syncSupport": "syncable"
	2. "url":  <YOUR_CORRECT_URL>	
6. Open wordpress.com, purchase a new domain and make it primary to your blog
7. Open Studio -> Sync tab
8. Assert that nothing changed in `appdata-v1.json`
9. Switch to this branch and open Sync tab again
10. Assert that you see:
	1. "syncSupport": "already-connected"
	2. "url":  <YOUR__DOMAIN_FROM_6-TH_STEP>
